### DISCUSSION TITLE
Bottom Tab Navigation Example Put Many Content/Navigation in Screen

### DISCUSSION THREAD

**Discussion by putuoka** [react-navigation/react-navigation#10731] [Q&A] [answered]
this link https://react-navigation-example.netlify.app/bottom-tabs is from example folder on react-navigation.

1. open the link
2. tap/click button "Replace With Feed"
3. tap/click button "GO BACK"

and the it will go back to the main directory list. What i mean is why it's back to main directory menu list not to lorem ipsum page inside the tab navigation. it should be still in tab navigation, right? How about if i want to add many content inside that screen that will navigate to many other navigation?

**Comment by satya164** [comment, marked as answer]
Because that screen doesn't exist anymore. It was replaced with the feed screen after clicking "replace with feed". If you push/navigate instead of replace then it'll go back to the screen.

**Reply by putuoka** [reply]
> Because that screen doesn't exist anymore. It was replaced with the feed screen after clicking "replace with feed". If you push/navigate instead of replace then it'll go back to the screen.

i mean how i can go back to the "lorem ipsum" page after clicking "Replace With Feed" Button? there is no option for doing that.
btw i appreciate your answer and accept it because after i explore more the page, now i understand an example already cover all function of bootom tab navigation. i wish lol because i really depend on this example. no other example on github all other is outdate lol

**Reply by satya164** [reply]
> i mean how i can go back to the "lorem ipsum" page after clicking "Replace With Feed" Button

You can't go back to it because you clicked "replace" which replaces the screen - the screen is not there anymore. If you want to open that screen again then you can do so from the main page.

If you don't want this behavior in your app then you would just use navigate instead of replace. This is an example app with various scenarios - the navigate scenario is on the next screen.

**Reply by putuoka** [reply]
> > i mean how i can go back to the "lorem ipsum" page after clicking "Replace With Feed" Button
> 
> You can't go back to it because you clicked "replace" which replaces the screen - the screen is not there anymore. If you want to open that screen again then you can do so from the main page.
> 
> If you don't want this behavior in your app then you would just use navigate instead of replace. This is an example app with various scenarios - the navigate scenario is on the next screen.

yes i got your point. Thank you.
